database/data_seeding/nosql_seeder.js

```diff
@@ -0,0 +1,48 @@
+const { user, password } = require('../../database_configs/nosql_database.config.js');
+const nano = require('nano')(`http://${user}:${password}@localhost:5984`);
+const { generateData } = require('./data_generator.js');
+
+const destroyDb = async (dbName) => {
+  await nano.db.destroy(dbName);
+}
+
+const createDb = async (dbName) => {
+  await nano.db.create(dbName)
+    .then((res) => {
+      console.log('Database successfully created', res);
+    })
+    .catch((err) => {
+      console.log('Error creating database', err);
+    });
+}
+
+const insertData = (data) => {
+  // code
+}
+
+const seedNoSqlData = async (numRecords, pace) => {
+
+  await destroyDb('product_features');
+
+  await createDb('product_features');
+
+  const pf = nano.use('product_features');
+
+  // while (numRecords < 0) {
+  //   // seed
+  // }
+  const data = generateData(5, 1, 'nosql');
+
+  data.map((record) => {
+    const { productIdEncid } = record;
+    pf.insert(record, productIdEncid)
```

Comment:
so in this function, you are mapping over all the data and adding each record one at a time. Which is probably fine for small data sets. But did you see the "nano.bulk({docs: dataToInsert})" method on the docs? It is used for inserting an array of docs.